app/workers/mhv/account_statistics_job.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module MHV
+  class AccountStatisticsJob
+    include Sidekiq::Worker
+
+    sidekiq_options retry: false
+
+    def perform
+      stats = { accounts_created_count: accounts_created.count,
+                existing_accounts_upgraded_count: existing_accounts_upgraded.count,
+                created_failed_upgrade_count: created_failed_upgrade.count,
+                created_and_upgraded_count: created_and_upgraded.count }
+      logger.info(mhv_account_statistics: stats)
+    end
+
+    private
+
+    def accounts_created
+      MhvAccount.where.not(registered_at: nil)
+    end
+
+    def existing_accounts_upgraded
+      MhvAccount.where(registered_at: nil).where.not(upgraded_at: nil)
+    end
+
+    def created_failed_upgrade
+      accounts_created.where(account_state: :upgrade_failed)
+    end
+
+    def created_and_upgraded
+      accounts_created.where.not(upgraded_at: nil)
+    end
```

Comment:
NITPICK: should these scopes exist maybe on the MhvAccount model?

Reply:
can we also add `failed_create` and a total `MhavAccount.count` as well? combinations of these should add up to the total.

Reply:
Could you also request that Wyatt or Craig run these for production so we can validate them and ensure they are meeting our expectations before merging?

Reply:
Is `failed_create` the same as `register_failed` (I don't see anything explicitly called failed_create).. here is my stab at a scope for it.. let me know if that seems right
```ruby
scope :failed_create, -> { where(registered_at: nil, account_state: :register_failed) }
```